crates/ruma-common/src/identifiers/room_alias_id.rs

```diff
@@ -37,6 +37,9 @@ impl RoomAliasId {
     }
 
     /// Create a `matrix.to` URI for an event scoped under this room alias ID.
+    ///
+    /// This is deprecated because room aliases are mutable, so the URI might break after a while.
+    #[deprecated = "Since Matrix 1.11, use `RoomId::matrix_to_event_uri` instead."]
```

Comment:
```suggestion
    #[deprecated = "Use `RoomId::matrix_to_event_uri` instead."]
```

The version of Matrix that added the deprecation doesn't really matter here, right?

Reply:
Agreed.